docs/documentation/sbom-guide.md

```diff
@@ -2,425 +2,239 @@
-  -S, --silent           silent mode, don't show progress spinner, but it will still output the result
-      --verbose          if true, print additional information
-  -v, --version          version for cas
-
-Use "cas [command] --help" for more information about a command.
-
-```
-
-#### Example on *Windows*
-
-##### Install:
-
-For *Windows* users, download your `cas` tool binary [here](https://github.com/codenotary/cas/releases/latest).
-
-##### Test run:
-
- Run the `.exe` file from the command line:
-```
-C:\Users\auser>cas-v1.0.3-windows-amd64.exe help
-Usage:
-  cas [command]
-
-Available Commands:
-  authenticate Authenticate assets against CAS
-  bom          Collect BOM information
-  completion   Generate the autocompletion script for the specified shell
-  help         Help about any command
-  inspect      Returns the asset history with low-level information
-  list         Returns the history of operations made with API key
-  login        Log in to Community Attestation Service
-  logout       Logout the current user
-  notarize     Notarize an asset onto Community Attestation Service
-  unsupport    Unsupport an asset
-  untrust      Untrust an asset
-
-Flags:
-      --caspath string   config files (default is user home directory)
-  -h, --help             help for cas
-  -o, --output string    output format, one of: --output=json|--output=''
-  -S, --silent           silent mode, don't show progress spinner, but it will still output the result
-      --verbose          if true, print additional information
-  -v, --version          version for cas
-
-Use "cas [command] --help" for more information about a command.
-```
-
-#### Alternative installation method for AlmaLinux systems
-
-An alternative to *Official CAS installation method* is to install the *CAS* tool package from the AlmaLinux repository. This will (obviously) work exclusively on the AlmaLinux system.
-
-:::tip
-The *CAS* package is available in AlmaLinux Architect's COPR (Community Owned Package Repository).  Here, we install it on system based on [AlmaLinux 9.1](https://wiki.almalinux.org/release-notes/9.1.html) release.
-:::
-     
-##### 1. Enable the COPR repo for codenotary package
-```
-$ sudo dnf copr --hub build.almalinux.org enable andrewlukoshko/codenotary
-```
+To access the immudb SBOM information for AlmaLinux you need an internet connection as records are stored in the AlmaLinux `git` repository.
+You also need to install the AlmaLinux SBOM tool:
+* Clone the [alma-sbom](https://github.com/AlmaLinux/alma-sbom) repository. 
+* Navigate to *alma-sbom* directory.
+* Create a Python Virtual Environment:
+  ```
+  python3.9 -m venv env
+  ```
+* Activate the Virtual Environment:
+  ```
+  source env/bin/activate
+  ```
+* Install dependencies:  
+  ```
+  pip install .
+  ```
+  Among installed dependencies there's [immudb-wrapper](https://github.com/AlmaLinux/immudb-wrapper) that is used to get SBOM information based on the rpm package hash.
```

Comment:
maybe something like: `there's immudb-wrapper, that is used to interact with our immudb database to save or get SBOM information`